### ISSUE TITLE
Resulting checkpoints .pt can't be used in AUTOMATIC1111 web ui

### ISSUE THREAD

**Issue by rkfg** [d8ahazard/sd_dreambooth_extension#962]
### Kindly read the entire form below and fill it out with the requested information.

**Please find the following lines in the console and paste them below. _If you do not provide this information, your
issue will be automatically closed._**

```
Python 3.10.9 (main, Dec  7 2022, 13:47:07) [GCC 12.2.0]
Commit hash: cc1c0554645acf5092e684dd970c3ed48c2a6f5a                                                                                                                                 
Installing requirements for Web UI        
                                                                                           
#######################################################################################################
Initializing Dreambooth                                                                                                                                                               
If submitting an issue on github, please provide the below text for debugging purposes:                                                                                               

Python revision: 3.10.9 (main, Dec  7 2022, 13:47:07) [GCC 12.2.0]
Dreambooth revision: 43ae9d55531004f1dedaea7ac2443e9b16739913
SD-WebUI revision: cc1c0554645acf5092e684dd970c3ed48c2a6f5a

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.17.dev451 installed. 
[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.
```

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.**
OK

**Describe the bug**

This is a crosspost for visibility, the currently generated `.pt` files can't be used in webui because of an unexpected file structure. Probably it's better to make them compatible so they can be used without merging with the base checkpoint. The original report: https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/7156 Using the generated file from built-in panel `Extra networks => Lora`, clicking the checkpoint there adds a tag to the prompt `<lora:checkpoint_name:1>`, generating an image succeeds but it doesn't use the Lora at all (removing the tag doesn't affect the image as well).

**Provide logs**

```
activating extra network lora with arguments [<modules.extra_networks.ExtraNetworkParams object at 0x7fca932ab1c0>]: TypeError
Traceback (most recent call last):
  File "/.../stable-diffusion-webui/modules/extra_networks.py", line 75, in activate
    extra_network.activate(p, extra_network_args)
  File "/.../stable-diffusion-webui/extensions-builtin/Lora/extra_networks_lora.py", line 23, in activate
    lora.load_loras(names, multipliers)
  File "/.../stable-diffusion-webui/extensions-builtin/Lora/lora.py", line 151, in load_loras
    lora = load_lora(name, lora_on_disk.filename)
  File "/.../stable-diffusion-webui/extensions-builtin/Lora/lora.py", line 83, in load_lora
    sd = sd_models.read_state_dict(filename)
  File "/.../stable-diffusion-webui/modules/sd_models.py", line 224, in read_state_dict
    sd = get_state_dict_from_checkpoint(pl_sd)
  File "/.../stable-diffusion-webui/modules/sd_models.py", line 197, in get_state_dict_from_checkpoint
    pl_sd = pl_sd.pop("state_dict", pl_sd)
TypeError: pop expected at most 1 argument, got 2
```

**Environment**

What OS?

Debian testing

What GPU are you using?

3090 Ti

**Comment by 0217ljh** [comment]
Same problem.It is a very important problem that the .pt file can't be used in webui. very very not convenient. Please fix it!

**Comment by eddjrn** [comment]
Same problem, only merging with checkpoint, which results on on a full size file, more or less with the same disadvantage as a full dream booth checkpoint

**Comment by edkenne** [comment]
Same issue on linux. Other PT files generated with other methods work, as do safetensors files.

**Comment by AbyszOne** [comment]
Several weeks, and still this A1111 extension make things that cannot be used in A1111. Confusing.

**Comment by enkynakamura** [comment]
Same here, same error reported

  File "E:\stable-diffusion-webui\modules\extra_networks.py", line 75, in activate
    extra_network.activate(p, extra_network_args)
  File "E:\stable-diffusion-webui\extensions-builtin\Lora\extra_networks_lora.py", line 23, in activate
    lora.load_loras(names, multipliers)
  File "E:\stable-diffusion-webui\extensions-builtin\Lora\lora.py", line 151, in load_loras
    lora = load_lora(name, lora_on_disk.filename)
  File "E:\stable-diffusion-webui\extensions-builtin\Lora\lora.py", line 83, in load_lora
    sd = sd_models.read_state_dict(filename)
  File "E:\stable-diffusion-webui\modules\sd_models.py", line 214, in read_state_dict
    sd = get_state_dict_from_checkpoint(pl_sd)
  File "E:\stable-diffusion-webui\modules\sd_models.py", line 187, in get_state_dict_from_checkpoint
    pl_sd = pl_sd.pop("state_dict", pl_sd)
**TypeError: pop expected at most 1 argument, got 2**

**Comment by Gushousekai195** [comment]
Same problem. FIX PLEASE!

**Comment by mlot** [comment]
Same issue here, lora pt file generated can not be applied in A1111, which lose most of lora's value.

**Comment by JackEllie** [comment]
I made this [LoRA-EXTRACTOR](https://github.com/JackEllie/LoRA-EXTRACTOR)from [sashaok123](https://github.com/sashaok123)
/[LoRA-EXTRACTOR](https://github.com/sashaok123/LoRA-EXTRACTOR)
I made a little modification, and I can re-extract LoRA from the generated checkpoint so that A1111 can read it, but I think this is not a good way. Can someone tell me how to merge lora.pt and lora_txt.pt into lora..safetensors? , I think it would be great to add this feature to the extension, I would love to eat and develop but I'm still a novice.
